### ISSUE TITLE
Hypervolt Session Energy Total Increasing

### ISSUE THREAD

**Issue by matttheotter** [gndean/home-assistant-hypervolt-charger#75]
Hi @gndean 

Regarding "Hypervolt Session Energy Total Increasing" it tracks as Unknown before and after sessions, is it possible that rather than unknown it could default to 0?

Thanks,

Matt

**Comment by silkit** [comment]
I was about to raise this as an issue as i have the same problem.
Although would it be better to show the end value/running total that resets at the end of the day?
Particularly if you are using Octopus Intelligent slots as the session energy will have several separate values for the different slots that Octopus give you so i would think you need some form of combined/cumulative value.

**Comment by matttheotter** [comment]
I setup a Utility Meter to do just that 🙂

**Comment by silkit** [comment]
;-) Thanks. I have set up an utility meter as well.
But the total increasing entity i think was supposed to do effectively create an equivalent for you.
[This is the reason the sensor was created in the first place](https://github.com/gndean/home-assistant-hypervolt-charger/issues/5)
It looks like the sensor was originally designed to show the daily cumulative as it was working like that, but i think it must have broken when there was a recent HA update which i think had some changes to sensor values in it.
It should probably be looked to make it easier for HA noobs that don't or can't set up things like utility meters lol

**Comment by matttheotter** [comment]
Yeah I did two meters a daily and a all time, there's no such thing as too much data 🤣

**Comment by emmet02** [comment]
Yeah having same issue here - "Unknown" is only value in town for the "Hypervolt Session Energy Total Increasing" sensor these days. 

Is there something we can fix inside the integration or do we need to go manual (utility) ?

**Comment by gndean** [comment]
I'm unsure how to handle this and thinking that a utility meter might be the only way.

For example, this is what I see with my car, controlled via Octopus Intelligent Go. This is the Hypervolt Session Energy sensor (not the total increasing one) and it resets to 0 at midnight. The car actually stops then restarts charging around midnight. What should the Hypervolt Session Energy Total Increasing sensor do in this example?

![image](https://github.com/user-attachments/assets/7870a382-e2ed-4a07-9d93-224c01626a1a)

**Comment by gndean** [comment]
Closing as a Utility Meter seems like the solution here and I'm not clear on what other options there are.
